apps/web/src/pages/get-started/consts/InAppUseCase.const.tsx

```diff
@@ -10,9 +15,20 @@ export const InAppUseCaseConst: OnboardingUseCase = {
     {
       title: 'Configure In-App provider',
       Description: function () {
+        const { integrations } = useGetIntegrationsByChannel({ channelType: ChannelTypeEnum.IN_APP });
+
+        const getInAppIntegrationUrl = () => {
+          const inAppIntegration = integrations?.[0];
+          if (!inAppIntegration) {
+            return ROUTES.INTEGRATIONS;
```

Comment:
in this case, I think we can open the Select provider sidebar with a preselected In-App tab... this URL should do that `/integrations/create?scrollTo=in_app`

Reply:
Sounds good, thanks!